README.md

```diff
@@ -56,6 +56,8 @@ env:
         fieldPath: metadata.uid
 ```
 
+* `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` can be used by the statsd client to globally set `{env, service, version}` as tags for all data emitted. The values of these tags will not override any other configured tags for `{env, service, version}`.
```

Comment:
I think we should rename the `will not override` since the code will actually send multiple value for the same tag if both set in the ENV and in the `Tags`.

Reply:
:+1: I've updated the docs